pkg/server/testdata/machine-configs/test-config.yaml

```diff
@@ -148,9 +148,9 @@ spec:
               EnvironmentFile=-/etc/kubernetes/kubelet-workaround
               ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
               ExecStartPre=/bin/mkdir --parents /etc/kubernetes/checkpoint-secrets
-              ExecStartPre=/bin/mkdir --parents /etc/kubernetes/cni/net.d
+              ExecStartPre=/bin/mkdir --parents /etc/cni/net.d
```

Comment:
can you not change this file, this is a testdata file for machine-config-server; not related to this change